lib/logger.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Utility functions for command line output logging from Protractor.
+ * May be used in different processes, since the launcher spawns
+ * child processes for each runner.
+ *
+ * All logging directly from Protractor, its driver providers, or runners,
+ * should go through this file so that it can be customized.
+ */
+export module Logger {
```

Comment:
look around for another way to export - if you are stuck please ping me on IM and I'll patch your change locally and confirm if I'm right :)